test/AutoinstSoftware_test.rb

```diff
@@ -52,7 +52,7 @@
 
       it "the packages are not added" do
         Yast::AutoinstSoftware.add_additional_packages(pkgs)
-        expect(Yast::PackagesProposal.GetResolvables("NetworkManager", :package))
+        expect(Yast::PackagesProposal.GetResolvables("NetworkManager", :package, optional: true))
```

Comment:
Is the `optional` needed here? I mean if that parameter is omitted, doesn't `GetResolvables` return a full list with all the resolvables including optional and mandatory? I'm honestly asking out of curiosity, I don't know how `GetResolvables` works.

Reply:
Good question. It is what I would have expected, but it does not work in that way:

```ruby
Yast.import "PackagesProposal"
Yast::PackagesProposal.AddResolvables("autoyast", :package, ["dummy-optional"], optional: true)
Yast::PackagesProposal.AddResolvables("autoyast", :package, ["dummy"])
Yast::PackagesProposal.AddResolvables("autoyast", :package, ["dummy-mandatory"], optional: false)
Yast::PackagesProposal.GetResolvables("autoyast", :package) #=> ["dummy", "dummy-mandatory"]
Yast::PackagesProposal.GetResolvables("autoyast", :package, optional: false) #=> ["dummy", "dummy-mandatory"]
Yast::PackagesProposal.GetResolvables("autoyast", :package, optional: true) #=> ["dummy-optional"]
```